### PR TITLE
LPS-130936

### PR DESCRIPTION
**Pull request by brianikim**
Resending per requested by https://github.com/brianchandotcom/liferay-portal/pull/101138#issuecomment-827565874

https://issues.liferay.com/browse/LPS-130936

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

**Comment by brianikim** [comment]
Hello @jkappler ,

Any update on this PR?

Sincerely,
Brian Kim

**Comment by jkappler** [comment]
Just started reviewing :)

File: portal-impl/src/com/liferay/portal/service/impl/GroupLocalServiceImpl.java [hunk 1]
```diff
@@ -299,6 +299,15 @@ public Group addGroup(
 			}
 		}
```
Comment (jkappler) [inline portal-impl/src/com/liferay/portal/service/impl/GroupLocalServiceImpl.java:301]: Hi @brianikim, you are only taking into account LocaleUtil.getDefault() when replacing the groupKey, even thought we retrieve it in different ways in the preceding if statement. Wouldn't it be better to just trim all the content in the name map, just to be sure that we don't leave anything out?

Also, was on PTO these last days, so I couldn't take a look at it

**Comment by jkappler** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/impl/GroupLocalServiceImpl.java (hunk 1 above)
Comment (brianikim) [reply portal-impl/src/com/liferay/portal/service/impl/GroupLocalServiceImpl.java:301]: Hello @jkappler ,

I agree with you. I have updated the PR with the latest changes reflected.

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
Just started reviewing :)

**Comment by jkappler** [comment]
Hi @brianikim, because of an issue in the latest commit I cannot test this, so we'll wait until it's fixed (shouldn't take long). I also added the trim to updateGroup, to cover all cases.

**Comment by jkappler** [comment]
https://github.com/liferay-echo/liferay-portal/pull/4458

**Pull request closed without merging**